vendor/github.com/buger/jsonparser/Makefile

```
@@ -0,0 +1,36 @@
+SOURCE = parser.go
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)